spec/features/tslr_claim_spec.rb

```diff
@@ -159,4 +159,115 @@
     visit claim_path("qts-year")
     expect(page).to have_current_path(root_path)
   end
+
+  context "editing a response" do
+    let(:claim_school) { create(:school, :tslr_eligible, name: "Claim School") }
+    let(:current_school) { create(:school, :tslr_eligible, name: "Current School") }
+
+    let(:claim) do
+      claim = create(:tslr_claim,
+        :eligible_and_submittable,
+        employment_status: :different_school,
+        claim_school: claim_school)
+      claim.current_school = current_school
+      claim.save
+      claim
+    end
+
+    before do
+      allow_any_instance_of(ClaimsController).to receive(:current_claim) { claim }
+      visit claim_path("check-your-answers")
+    end
+
+    scenario "Teacher can edit a field" do
+      find("a[href='#{claim_path("national-insurance-number")}']").click
+
+      fill_in "National Insurance number", with: "AB123456C"
+      click_on "Continue"
+
+      expect(claim.reload.national_insurance_number).to eq("AB123456C")
+      expect(page).to have_content("Check your answers before sending your application")
+    end
+
+    scenario "Teacher sees their original current school when editing" do
+      find("a[href='#{claim_path("current-school")}']").click
+
+      expect(find("input[name='school_search']").value).to eq(current_school.name)
+    end
+
+    context "Teacher goes to change their claim school" do
+      before do
+        find("a[href='#{claim_path("claim-school")}']").click
+      end
+
+      scenario "sees their original claim school" do
+        expect(find("input[name='school_search']").value).to eq(claim.claim_school.name)
+      end
+
+      context "changes their claim school" do
+        before do
+          choose_school schools(:penistone_grammar_school)
+        end
+
+        scenario "school is changed correctly" do
+          expect(claim.reload.claim_school).to eql schools(:penistone_grammar_school)
+        end
+
+        scenario "Teacher is redirected to the are you still employed screen" do
+          expect(current_path).to eq(claim_path("still-teaching"))
+        end
+
+        context "is still teaching at the same school" do
+          before do
+            choose_still_teaching
+          end
+
+          it "current school is set correctly" do
```

Comment:
Should these be `scenario`s rather than `it`s?

Reply:
They're functionally the same, but I'd say `it` is more expressive when you're only checking the state of something

Reply:
On further inspection, it seems like the guidance is to use `scenario` in all cases, so happy to change this for consistency 👍